test-cases/manager/manager-backup-and-restore-5TB-single-node.yaml

```diff
@@ -0,0 +1,28 @@
+test_duration: 4320
+
+prepare_write_cmd: ["cassandra-stress write cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..1342177280",
```

Comment:
I think it would make more sense to split this test into 2 tests, one test that creates the keyspace and do the backup.
(with option to persist the backup for later usages (like you do in tests above).

And another test, that only restore from a previous backup to a clean cluster.

This will improve things by:
1. Allow you to run only one test if you need (e.g. problem with backup, so you run only this one)
2. It will cut the time to run the test by ("half") because you can run them both in parallel.
3. In most cases we won't even run the 5tb backup test, but only the restore.
4. the restore test can run on bigger cluster, while the backup is enough to run on single node cluster.